src/main/java/com/rybka/command/HistoryCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.rybka.command;
+
+import com.rybka.dao.CurrencyHistoryDAO;
+import com.rybka.exception.DBConnectionException;
+import com.rybka.model.CurrencyHistory;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j;
+
+import java.util.List;
+
+@Log4j
+@RequiredArgsConstructor
+public class HistoryCommand implements Command {
+    private final CurrencyHistoryDAO currencyHistoryDAO;
+
+    @Override
+    public void execute() {
+        try {
+            var histories = history();
+            print(histories);
+        } catch (Exception exception) {
+            log.error("Some issues have been occurred while returning history. Reason: " + exception.getMessage());
+        }
+    }
+
+    public List<CurrencyHistory> history() {
```

Comment:
I think you don't need this method. Put the code into the `execute` method.

Reply:
fixed